app/api/v1/endpoints/enterprise_health.py

```
@@ -0,0 +1,322 @@
+            status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
+            detail="Service temporarily unavailable"
+        )
+
+
+@router.get("/health/detailed",
+           response_model=DetailedHealthResponse,
+           dependencies=[Depends(require_role([UserRole.ADMIN]))],
+           tags=["Health"],
+           summary="Detailed health and monitoring data")
+async def detailed_health_check(
+    current_user: User = Depends(require_role([UserRole.ADMIN]))
```

Ruff (0.12.2)

Unused function argument: `current_user`

(ARG001)